tests/mount_test.go

```diff
@@ -23,36 +69,54 @@ func TestSingleMount(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "moar", out)
 
+	// we shadow mnt/m1 with a mount
 	out, err = ts.run("init --store mnt/m1 --path " + ts.storeDir("m1") + " --rcs=noop " + keyID)
 	t.Logf("Output: %s", out)
 	require.NoError(t, err)
 
-	out, err = ts.run("show mnt/m1/secret")
+	// check the mount is there
+	out, err = ts.run("mounts")
+	assert.NoError(t, err)
+	want := "gopass (" + ts.storeDir("root") + ")\n"
+	want += "└── mnt\n    └── m1 (" + ts.storeDir("m1") + ")"
+	assert.Equal(t, strings.TrimSpace(want), out)
+
+	// check that the mount is not containing our shadowed secret
+	out, err = ts.run("show -f mnt/m1/secret")
 	assert.Error(t, err)
 	assert.Equal(t, "\nError: failed to retrieve secret 'mnt/m1/secret': Entry is not in the password store\n", out)
 
-	ts.initSecrets("mnt/m1/")
+	// insert some secret at the place that is shadowed by the mount
+	_, err = ts.runCmd([]string{ts.Binary, "insert", "mnt/m1/secret"}, []byte("food"))
+	require.NoError(t, err)
 
+	// check that the mount is containing our new secret shadowing the old one
+	out, err = ts.run("show -f mnt/m1/secret")
+	assert.NoError(t, err)
+	assert.Equal(t, "food", out)
+
+	// check that the mount is listed
 	list := `gopass
+├── baz
 ├── fixed
 │   ├── secret
 │   └── twoliner
 ├── foo
 │   └── bar
-├── mnt
+└── mnt
 `
-	list += "│   └── m1 (" + ts.storeDir("m1") + ")\n"
-	list += `│       ├── fixed
-│       │   ├── secret
-│       │   └── twoliner
-│       ├── foo
-│       │   └── bar
-│       └── baz
-└── baz`
+	list += "    └── m1 (" + ts.storeDir("m1") + ")\n"
+	list += `        ├── baz
+        ├── fixed
+        │   ├── secret
+        │   └── twoliner
+        └── foo
+            └── bar`
 
 	out, err = ts.run("list")
 	assert.NoError(t, err)
-	t.Skip("integration test seems broken, but not manual tests")
+
+	t.Skip("this test is currently failing because the list is shadowing the mount")
```

Comment:
This is referencing #1460 I guess?
In that case the skipping is good until we fix it, but the reason seems to be odd (better than the previous one, but still). Shouldn't it say `this test is currently failing because the list command is not properly shadowing the mount`?